docker-compose.yml

```diff
@@ -1,26 +1,30 @@
 version: "3"
-
+volumes:
+  db:
 services:
   db:
     image: mariadb
     restart: always
     ports:
       - "3306:3306"
     environment:
+      - MYSQL_DATABASE=iabot
       - MYSQL_PASSWORD=5yBtFxeedrfupieNk7mp1oWyP8aR91kAp9nO8oSH
       - MYSQL_ROOT_PASSWORD=5yBtFxeedrfupieNk7mp1oWyP8aR91kAp9nO8oSH
     volumes:
-      - ./mariadb:/var/lib/mysql
-  web:
-    image: phpstorm/php-73-apache-xdebug-27
+      - db:/var/lib/mysql
+  iabot:
+    build: .
     ports:
       - "8080:80"
     volumes:
       - ./app/src:/var/www
-      - ./vendor:/vendor
```

Comment:
I would retain this.  You can attach external IDEs to this containers and it may be useful to have to step into a composer library for debugging purposes.

Reply:
Fixed